### ISSUE TITLE
NTR: Pelvic imaging: fallopian tube hydrosalpinx  (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9036]
New term request Pelvic imaging: fallopian tube hydrosalpinx  (elthat_200406050959):   UMLS: 0221376

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000146
- label: Hydrosalpinx
- parents: [Abnormal fallopian tube morphology (HP:0011027)](https://hpo.jax.org/app/browse/term/HP:0011027)
- pmid: PMID:30824209
- def: Hydrosalpinx is a distension or dilatation of the fallopian tube in the presence of a distal tubal occlusion.
- comment: The most common cause of hydrosalpinx is pelvic inflammatory disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9036
- synonyms:
